src/Main.java

```diff
@@ -1,5 +1,99 @@
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача номер 1:");
+        byte a = 25;
+        short b = 250;
+        int c = 2500;
+        long d = 25000L;
+        float e = 25.5f;
+        double g = 225.5;
+        System.out.println("1)Значение переменной <a> с типом <byte> равно " + a);
+        System.out.println("2)Значение переменной <b> с типом <short> равно " + b);
+        System.out.println("3)Значение переменной <c> с типом <int> равно " + c);
+        System.out.println("4)Значение переменной <d> с типом <long> равно " + d);
+        System.out.println("5)Значение переменной <e> с типом <float> равно " + e);
+        System.out.println("6)Значение переменной <g> с типом <double> равно " + g);
+
+        System.out.println("Задача номер 2:");
+        float z = 27.12f;
+        long y = 987678965549L;
+        double x = 2.786;
+        short w = 569;
+        short v = -159;
+        int t = 27897;
+        byte s = 67;
+
+        System.out.println("Задача номер 3:");
+        byte studentsLudPav = 23;
+        byte studentsAnnSer = 27;
+        byte studentsEktAndr = 30;
+        short paper = 480;
+        int studentsWorksheets = paper / (studentsEktAndr + studentsAnnSer + studentsLudPav);
+        System.out.println("На каждого ученика рассчитано " + studentsWorksheets + " листов бумаги.");
+
+        System.out.println("Задача номер 4:");
+        byte outputPerMinute = 16;
```

Comment:
```suggestion
        byte outputPerMinute = 16 / 2;
```
в минуту 16 / 2, а не 8 (см. задание)